api/tests/routes/backoffice/admin_test.py

```diff
@@ -103,6 +103,55 @@ def test_update_role_with_empty_permissions(self, authenticated_client):
         db.session.refresh(role)
         assert role.permissions == []
 
+    def test_log_update_role(self, authenticated_client):
+        perms = perm_factories.PermissionFactory.create_batch(4)
+        old_perms = [perms[0], perms[1]]
+        role_to_edit = perm_factories.RoleFactory(permissions=old_perms)
+        role_not_to_edit = perm_factories.RoleFactory(permissions=old_perms)
+
+        new_perms = [perms[1], perms[3]]
+        base_form = {}
+        for perm in perms:
+            base_form[perm.name] = perm in new_perms
+
+        response = self.post_to_endpoint(authenticated_client, role_id=role_to_edit.id, form=base_form)
+        assert response.status_code == 303
+
+        action = history_models.ActionHistory.query.one()
+        assert action.actionType == history_models.ActionType.ROLE_PERMISSIONS_CHANGED
+        assert action.authorUserId is not None
```

Comment:
```suggestion
        assert action.authorUserId == legit_user.id
```
en ajoutant au prototype de la méthode : 
`def test_log_update_role(self, legit_user, authenticated_client):`

Reply:
La _fixture_ `legit_user` est définie ici : 
https://github.com/pass-culture/pass-culture-main/blob/v266.0.4/api/tests/routes/backoffice/conftest.py#L164
donc en gros cet objet est créé pour le test lorsqu'il est passé en paramètre.
C'est cet utilisateur qui est authentifié dans le BO lorsque tu fais la requête à partir d'`authenticated_client` (_fixture_ juste en-dessous), donc cet utilisateur est l'auteur de toutes les actions par ce "client".